Library/Homebrew/global.rb

```diff
@@ -124,6 +124,10 @@ def owner_uid
     def running_as_root_but_not_owned_by_root?
       running_as_root? && !owner_uid.zero?
     end
+
+    def auto_update_command?
+      ENV.fetch("HOMEBREW_AUTO_UPDATE_COMMAND", false).present?
+    end
```

Comment:
Should this go in `EnvConfig`?

Reply:
I debated that. It's not really a product of a user configuration though so didn't feel right. Doesn't feel like it belongs anywhere obviously.